server/graphql/types/src/types/program/encounter.rs

```diff
@@ -247,7 +247,7 @@ pub struct SuggestedNextEncounterNode {
 #[Object]
 impl SuggestedNextEncounterNode {
     async fn start_datetime(&self) -> DateTime<Utc> {
-        DateTime::<Utc>::from_utc(self.suggested.start_datetime, Utc)
```

Comment:
This was a warning I introduced recently and missed because of all the other warnings...